pages/api/study/data/[key].js

```diff
@@ -0,0 +1,24 @@
+import { useCors } from '@lib/cors'
+import dbConnect from '@lib/db-connect'
+import Study from '@models/study'
+
+// GET /study/data/[key]
```

Comment:
change to /study/metadata or /study/config (better option). Doesn't make sense for GET /study/data to get controls and song while POST /study/data uploads participant data